src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -261,7 +272,11 @@ export function updateLayeredMaterialNodeImagery(context, layer, node) {
                 // and stop retrying after X attempts.
             }
 
-            node.layerUpdateState[layer.id].success();
+            if (!failureParams) {
+                node.layerUpdateState[layer.id].success();
+            } else {
+                node.layerUpdateState[layer.id].retry(Date.now(), { targetLevel: failureParams.targetLevel });
```

Comment:
Why do you use `retry()` on success?

Reply:
if there was a previous failure, a loading success doesn't mean that the best texture is loaded
i add a comment

Reply:
`sucess()` doesn't imply that it will not try to improve the texture later.

Reply:
ok